### PR TITLE
Document taas

### PR DESCRIPTION
**Pull request by liapretorius**
There is more that could be added to this in terms of setting up a Framework to work within TaaS. I figured we can iterate on this so long has everything here is accurate and complete in what it is describing and it's generic enough to be used by any company who uses Akkeris with TaaS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,104 @@
...
+
+-   `aka update`
+
+  <!-- ToDo: Information regarding setting up a TaaS app from which to run tests -->
+
+Install TaaS plugin
+
+-   There is no UI for TaaS currently.  From command line type `aka`.  TaaS commands are at the bottom of the list if you have it installed.
+
+-   If not installed, type `aka plugins:install TaaS`
```
Comment (murrayres) [inline README.md]: probably should not be camel case.

File: README.md [hunk 2]
```diff
@@ -1 +1,104 @@
...
+	-   `aka apps:info –a <app_name>`
+
+	-   Pipeline name will be listed if it exist
+	-   Space_name = everything after the first dash in the app_name
+
+  
+
+Register tests
+
+-   Use Akkeris TaaS plugin to register tests, `aka TaaS:tests:register`
```
Comment (murrayres) [inline README.md]: probably should not be camel case.

File: README.md [hunk 3]
```diff
@@ -1 +1,104 @@
...
+
+-   Enter values requested by plugin as indicated below
+
+	-   App Name = <app_name>, e.g. myapp-myspace-env
+	-   Job Name = <app_name>
+	-   Job Space = TaaS
```
Comment (murrayres) [inline README.md]: probably should not be camel case.

**Review by murrayres — COMMENTED** [review]
In general, if it's part of a command it will be "taas" instead of "TaaS"

File: README.md [hunk 4]
```diff
@@ -1 +1,104 @@
...
+	-   Timeout = how long before test run should timeout in seconds
+	-   Start delay = how many seconds before running tests (gauge based on how long app takes to load).
+	- 	Slack Channel (no leading #) = Slack channel where test results should be sent.
+	-   Environment variables = Optional.  Format is KEY=value.
+
+  
+
+-   Enter the rest of the env vars Metis needs to run the tests
+
+-   `aka TaaS:config:set KEY=value`
```
Comment (murrayres) [inline README.md]: probably should not be camel case.

File: README.md [hunk 5]
```diff
@@ -1 +1,104 @@
...
+  
+
+-   Enter the rest of the env vars Metis needs to run the tests
+
+-   `aka TaaS:config:set KEY=value` 
+-   For bulk loading of env vars you can use the following script from within the Metis root directory. Make sure your env files are up to date.
+
+-   `cat ./path/to/.env.file | grep = | awk '{print "aka TaaS:config:set <name-of-test> "$0}' | sed -e 's/ = /=/g' > temp.sh` 
+
+-   `aka TaaS:tests:info` to see all tests and find their names
```
Comment (murrayres) [inline README.md]: probably should not be camel case.

File: README.md [hunk 6]
```diff
@@ -1 +1,104 @@
...
+-   `chmod +x ./temp.sh`
+-   `./temp.sh`
+
+-   Check your work.
+	
+	-   `aka TaaS:tests:info <name_of_test>` to see details of registered test
+	-   `aka TaaS:tests:trigger <name_of_test>` to trigger a test run.
+	-   Check `your_designated_slack_channel` for results
+
+List of available TaaS commands
```
Comment (murrayres) [inline README.md:86]: probably should not be camel case.

Comment (liapretorius) [reply README.md:86]: Fixed all. Thanks for catching that.

**Review by murrayres — APPROVED** [review]

**Pull request merged**